pennylane_sf/_version.py

```diff
@@ -16,4 +16,4 @@
    Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.19.0-dev"
+__version__ = "0.19.0"
```

Comment:
Need to patch script to include a newline here, although this error does not arise in a different plugin, leading me to believe that code factor is inconsistent amongst plugins
```suggestion
__version__ = "0.19.0"

```